### DISCUSSION TITLE
Runtime configs not read in standalone mode

### DISCUSSION THREAD

**Discussion by bobbyphilip** [vercel/next.js#38224] [Help]
This is based on the discussion in https://github.com/vercel/next.js/issues/33910

There was a [suggestion]( https://github.com/vercel/next.js/issues/33910#issuecomment-1028081728) to  modify server.js to use conf from next.config.js
Shouldnt this be the default behaviour for someone enabling output:standalone?

**Comment by flextremedev** [comment]
It sounds to me they can't do that, because of possible dependencies inside next.config.js. However it would be great, if there was a way to pass in an optional conf path as a tracing option or an opt-in like "trace next.config.js" to use at your own risk.
